### DISCUSSION TITLE
Customizing install_manifest.txt location (or raising an error if not found)

### DISCUSSION THREAD

**Discussion by ihnorton** [scikit-build/scikit-build#997] [Ideas]
Re: https://github.com/scikit-build/scikit-build/pull/996

I think it would be useful to:
- raise an error (or at least a warning) if no `install_manifest.txt` is found in the `CMAKE_BUILD_DIR`, which can happen with eg a superbuild install. Considering this _the_ key link for Python packaging to pick up the skbuild/CMake artifacts, a missing install_manifest is likely a problem in most circumstances.
- possibly allow customization of where to look for the manifest.

If either/both of these are of interest, I can make a PR.

**Comment by henryiii** [comment]
I think we could recursively collect install_manifests?
